config/tests.sh

```diff
@@ -29,7 +29,12 @@ run_for_framework() {
       fi
 
     else
-      python -m pytest --durations=50 --html=$REPORT_DIR/report_$1.html -v -s --self-contained-html tests/$1
+      if [ "$1" = "tensorflow2" ] ; then
+        python -m pytest --durations=50 --html=$REPORT_DIR/report_$1/eager_mode.html -v -s --self-contained-html tests/$1
+        python -m pytest --durations=50 --non-eager --html=$REPORT_DIR/report_$1/non_eager_mode.html -v -s --self-contained-html tests/$1
```

Comment:
Relying on test execution order can be flaky. Better to reset things in the test code so tests can truly be run independently.

Reply:
Since we can't toggle between eager and non-eager execution, introduced this change. https://github.com/awslabs/sagemaker-debugger/pull/190/files#diff-484462fced51d1a06b1d93b4a44dd535R54.

Is it possible to reset? I couldn't much about this scenario in the tensorflow documentation or GitHub issue list.

This comment also gave https://github.com/awslabs/sagemaker-debugger/blob/989a4d15e17541b06027d6f658f1fd15fab63566/tests/tensorflow/keras/test_keras.py#L137 also gave me the impression that previous attempts were unsuccessful.

Reply:
also, this change is in this PR and #190 . will remove it from either depending on which gets merged first